css/styles.css

```diff
@@ -0,0 +1,43 @@
+/* below is the CSS for the header */
+
+header {
+  background-color: #4c2676;
+  color: white;
+}
+
+.padding-20 {
+  padding: 10px;
+}
```

Comment:
Why named `padding-20` when it's giving a padding of 10px? While yes it ends up being 20px on each side, but in total it would be 40px of padding. I would name this `padding-10` since it's 10px on each side.

Reply:
i forgot to change the name, original padding value was 20px.